src/modules/Rear.rb

```diff
@@ -59,26 +58,15 @@ def GetUsbPartitions
       usbdevs = []
       usbparts = {}
 
-      storage = Storage.ReReadTargetMap
-
-      Builtins.foreach(storage) do |device, devicemap|
-        if Ops.get_symbol(devicemap, "transport", :none) == :usb
-          parts = Ops.get_list(devicemap, "partitions", [])
-          Builtins.foreach(parts) do |part|
-            dev = Ops.get_string(part, "device", "")
-            size_k = Ops.get_integer(part, "size_k", 0)
-            Ops.set(
-              usbparts,
-              Builtins.sformat("usb://%1", dev),
-              Builtins.sformat(
-                "%1 (%2)",
-                dev,
-                Storage.KByteToHumanString(size_k)
-              )
-            )
-          end
+      storage = Y2Storage::StorageManager.instance
+      # (re)probing
```

Comment:
Maybe, simply to add a comment to clarify that this module is run standalone and it does not change staging.